api/v1/api.go

```diff
@@ -77,18 +81,160 @@ type postIsiExportResp struct {
 type IsiExport struct {
 	ID    int      `json:"ID"`
 	Paths []string `json:"paths"`
+	Clients []string `json:"clients"`
 }
 
 type getIsiExportsResp struct {
 	ExportList []*IsiExport `json:"exports"`
 }
 
+type isiThresholds struct {
```

Comment:
Sounds good. I'm guessing this "private-ness" is just a convention and not enforced by the language, kind of like in Python there's a common convention of naming private variables with underscore prefixes, right? So that being the case then I think the getIsiExportsResp struct needs to be renamed because I see it being returned by GetIsiExports, so I think that would make it a public struct.